types/features/features.go

```diff
@@ -1,122 +1,6 @@
-// Package features provides the JSON structure that is printed by `runc features` (since runc v1.1.0).
-// The types in this package are experimental and subject to change.
+// Package features provides the annotations for [github.com/opencontainers/runtime-spec/specs-go/features].
```

Comment:
This (the link) does not conform to godoc spec in https://go.dev/doc/comment#links

Reply:
Sorry, what am I missing?

Reply:
Sorry for the noise, I haven't realized this is a doc link, i.e. a link to the Go package. Everything is correct here.